.github/workflows/build-and-push-image.yml

```
@@ -2,54 +2,66 @@
+      
+      ## created at 20241220
+      - name: Create Sentry release
+        uses: getsentry/action-release@v1
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+          SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+          # SENTRY_URL: https://sentry.io/
+        with:
+          environment: developement
+          version : ${{ github.sha }}
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)